homeassistant/components/romy/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Sensor checking adc and status values from your ROMY."""
+
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    AREA_SQUARE_METERS,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    EntityCategory,
+    UnitOfLength,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import RomyVacuumCoordinator
+from .entity import RomyEntity
+
+SENSORS: list[SensorEntityDescription] = [
+    SensorEntityDescription(
+        key="battery_level",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.BATTERY,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
```

Comment:
Maybe disable this one by default

Reply:
entity_registry_enabled_default=False added for all

Reply:
enabled it now per default again, why would like to have it disabled per default?
cause enable every sensor separately is tedious job,
(didn't see that on testing before cause i didn't do factory reset)

Reply:
Typically entities which are not used so much should be set as disabled and the users who wish can enable them. 
I didn't say for all, just for this one as it's quite uncommon `rssi` provides any value.

There is also possible to mass-enable entities so no, it's not a very tedious task.

Reply:
nice, wasn't aware you can set all at once
makes sense, most users don't care about rssi, disabled it now per default again